packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -209,17 +224,19 @@ export const PaymentUpdateForm = ({
     <section className="settings-unit" aria-labelledby="payment-information">
       <div className="payment-update" data-testid="payment-update">
         {stripeSubmitInProgress && (
-          <AlertBar className="alert alertPending">
-            <Localized id="sub-route-idx-updating">
-              <span>Updating billing information...</span>
-            </Localized>
+          <AlertBar
+            className="alert alertPending"
+            dataTestId="alert-pending"
+            headerId={alertPendingAriaLabelledBy}
+            descId={alertPendingAriaDescribedBy}
+            localizedId="sub-route-idx-updating"
+          >
+            Updating billing information...
```

Comment:
Not sure if we have rules around `...` vs `…` (although I guess we previously used `...`, so 🤷) /cc @flodolo

Reply:
Copy should use typographic apostrophes, double quotes, and ellipsis (`’“”…` vs `'""...`).

Note that you can now flag @mozilla/fxa-l10n for questions, which is more scalable than flagging individuals ;-)